build/dependencies.props

```diff
@@ -0,0 +1,44 @@
+﻿<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  <!-- This files is typically managed by automation. Execute 'run.ps1 upgrade deps' to update these variables to the last-known-good versions. -->
+  <PropertyGroup Label="Package Versions">
+    <BenchmarkDotNetPackageVersion>0.10.9</BenchmarkDotNetPackageVersion>
+    <InternalAspNetCoreSdkPackageVersion>2.1.0-preview1-15544</InternalAspNetCoreSdkPackageVersion>
+    <LibuvPackageVersion>1.10.0</LibuvPackageVersion>
+    <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.1.0-preview1-27450</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
```

Comment:
Can't we have a shared version for AspNetCore things?

Reply:
While there is certainly duplication here, we've been burned in the past by trying to combine variables. Patch builds, one-off version changes, 'special packages', etc. There were enough edge cases that we found it's simpler and more reliable to do `packageId.ToPascalCase() + "PackageVersion"`.

Reply:
See https://github.com/aspnet/HttpClientFactory/pull/22/files#diff-30dcc9789f8ea3c7632e4f054e60c072. I think it's ridiculous...